code/modules/uplink/uplink_items.dm

```diff
@@ -1267,118 +1560,179 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
+			They allow you to see organisms through walls by capturing the upper portion of the infrared light spectrum, \
+			emitted as heat and light by objects. Hotter objects, such as warm bodies, cybernetic organisms \
+			and artificial intelligence cores emit more of this light than cooler objects like walls and airlocks."
+	item = /obj/item/clothing/glasses/thermal/syndi
+	cost = 4
 
-/datum/uplink_item/suits/turtlenck_skirt
-	name = "Tactical Skirtleneck"
-	desc = "A slightly armored suit that has no sensor on them, if someone sees you in this hope they think its a fake."
-	item = /obj/item/clothing/under/syndicate/skirt
+/datum/uplink_item/tools/nightvision
+	name = "Night Vision Goggles"
+	desc = "Simple goggles that allow one to see in the dark. Incredibly useful for infiltration."
+	item = /obj/item/clothing/glasses/night
 	cost = 1
-	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops) //They already get these
 
-/datum/uplink_item/suits/padding
-	name = "Soft Padding"
-	desc = "Padding to add to a jumpsuit to help against melee and bullets."
-	item = /obj/item/clothing/accessory/padding
+/datum/uplink_item/tools/magboots
+	name = "Blood-Red Magboots"
+	desc = "A pair of magnetic boots with a Syndicate paintjob that assist with freer movement in space or on-station \
+			during gravitational generator failures. These reverse-engineered knockoffs of Nanotrasen's \
+			'Advanced Magboots' slow you down in simulated-gravity environments much like the standard issue variety."
+	item = /obj/item/clothing/shoes/magboots/syndie
 	cost = 2
-	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/suits/kevlar
-	name = "Kevlar sheets"
-	desc = "Kevlar sheets to add to jumpsuit to help against bullets and melee."
-	item = /obj/item/clothing/accessory/kevlar
-	cost = 2
-	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+/datum/uplink_item/tools/combatbananashoes
+	name = "Combat Banana Shoes"
+	desc = "While making the wearer immune to most slipping attacks like regular combat clown shoes, these shoes \
+		can generate a large number of synthetic banana peels as the wearer walks, slipping up would-be pursuers. They also \
+		squeak significantly louder."
+	item = /obj/item/clothing/shoes/clown_shoes/banana_shoes/combat
+	cost = 6
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/suits/plastic
-	name = "Plastic sheet"
-	desc = "Plastic body sheet to add to a jumpsuit to help against laser and energy harm."
-	item = /obj/item/clothing/accessory/plastics
+/datum/uplink_item/tools/military_belt
+	name = "Chest Rig"
+	desc = "A robust seven-slot set of webbing that is capable of holding all manner of tactical equipment."
+	item = /obj/item/storage/belt/military
+	cost = 1
```

Comment:
Nukie only or make it cost like 3 or 5 this is REALLY good on what it holds

Reply:
It has always been 1 TC

Reply:
Its always been a nukie item I'd thought do to it holding anything thats small, making combos like blue space kit + LGMs to fit in or holding a legit hardsuit spare in it

Reply:
My argument with that is that belt items should never have been able to fit in bags to begin with, but I suspect a lot of people would get angry about it.

Reply:
No its the other way around, mil belts have no whitelist. So they fit ANYTHING small in it. Making the bluespace kit godly good for it as it will become a free shotgun holding kit.
Saw down riots and they fit.
shirnk down mag guns and they fit.
Its just godly good with lava land / station gear